server/pulp/server/managers/repo/importer.py

```diff
@@ -346,6 +346,32 @@ def set_importer_scratchpad(self, repo_id, contents):
         repo_importer['scratchpad'] = contents
         importer_coll.save(repo_importer, safe=True)
 
+    @staticmethod
+    def _sanitize_passwords(importer):
+        """
+        Return a sanitized importer.
+
+        This method replaces passwords with '*****'.
+
+        :return: a repo importer
+        :rtype:  pulp.server.db.model.repository.RepoImporter
+
+        """
+
+        # an empty record is already free of passwords:)
+        if not importer:
+            return
+
+        if 'config' in importer and importer['config']:
+            if 'proxy_password' in importer['config'] and \
+                    importer['config']['proxy_password'] is not None:
+                importer['config']['proxy_password'] = '*****'
+            if 'basic_auth_password' in importer['config'] and \
+                    importer['config']['basic_auth_password'] is not None:
+                importer['config']['basic_auth_password'] = '*****'
```

Comment:
I realized during breakfast that these should be using strings from `constants`. I will update the PR.

:egg: :coffee: